src/pages/docs/step-ca/configuration.mdx

```diff
@@ -238,6 +238,17 @@ CA's keys.
 
```

Comment:
Should we also update the example config to have a crl section with just `"enable": false` in it?

Reply:
sure. I've not seen an actual config using this stuff yet, so was hesitant to add anything to the example.